pkg/golinters/revive.go

```diff
@@ -388,3 +389,23 @@ func defaultConfig() *lint.Config {
 	}
 	return &defaultConfig
 }
+
+// TODO(ldez) https://github.com/golangci/golangci-lint/issues/2997
+func ignoreRules(conf *lint.Config) {
+	f := []string{
+		"context-keys-type",
+		"errorf",
+		"modifies-value-receiver",
+		"range-val-address",
+		"string-of-int",
+		"time-equal",
+		"time-naming",
+		"unexported-return",
+		"unhandled-error",
+		"var-declaration",
+	}
+
+	for _, s := range f {
+		delete(conf.Rules, s)
+	}
+}
```

Comment:
Maybe worth adding a log if the rule is present in the config so the user gets notified it's being disabled instead of just silently dropping it?

Reply:
PTAL